src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,20 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
-);
+export const CommentInfo = ({ comment }) => {
+  const { id, name, email, body } = comment;
+
+  return (
+    <>
+      <div key={id} className="CommentInfo">
```

Comment:
```suggestion
      <div className="CommentInfo">
```
Not all comments are fixed